src/datastore/pg-store.ts

```diff
@@ -4524,4 +4524,11 @@ export class PgStore extends BasePgStore {
     `;
     if (result.count) return result[0];
   }
+
+  async getLastEventObserverRequestSequenceId(): Promise<string | undefined> {
+    const result = await this.sql<{ sequence_id: string }[]>`
+      SELECT sequence_id FROM event_observer_requests ORDER BY id DESC LIMIT 1
```

Comment:
This might be super slow on mainnet without an index given the size of the table. Fine for now though

Reply:
No worries, `id` is the primary key of the table so this query ends up being super fast